docs/get-started/sumo-logic-ui.md

```diff
@@ -1,115 +1,135 @@
+    * **My Preferences**. Your account session settings.
 
-You can customize tabs by renaming them, then close them when they are no longer relevant.
+For more information, see [Account Preferences and Credentials](account-settings-preferences.md).
 
-To rename or close a tab, do the following:
+### Get Help: Docs, community, and more
 
-1. To rename a tab, double-click the name field, enter the new name, and press **Return**.
+There are a number of places where you can get the answers to questions you have:
 
-    ![WTS_UI_Tab-rename.png](/img/get-started/ui/WTS_UI_Tab-rename.png)
+* Check out our [Release Notes](/docs/release-notes)
+* Search [Sumo Docs](https://help.sumologic.com/)
+* Visit the **Learn Page** in the Sumo Logic UI
+* Post a question on the [Sumo Logic Community](https://support.sumologic.com/hc/en-us/community/topics)
+* Contact [Support](https://support.sumologic.com/)
+* Join our [Sumo Logic Slack](https://sumodojo.slack.com/) channel
 
-1. To close a tab, hover the cursor over the tab and click the X icon on the right.
+:::tip
+See [Help Resources and Contacts](/docs/get-started/help) for full information.
+:::
 
-    ![WTS_UI_Tab-delete.png](/img/get-started/ui/WTS_UI_Tab-delete.png)
+## Administrator tasks
 
-### Customize your environment with tab options
+### Manage data collection, data settings, and alerts
 
-Log Search, Metrics, and Live Tail tabs have additional options you can use to customize your environment. The tabs even stay open when you sign out and sign back in again, so you can start where you left off. Clicking any of the following icons opens a tabbed window.
+Sumo Logic Administrators (Admins) are responsible for managing data collection, data settings, and alerts for their organization. You must have Sumo Logic Admin role privileges to perform these tasks.
 
-![WTS_UI_LogSearch-Metrics-LiveTail.png](/img/get-started/ui/WTS_UI_LogSearch-Metrics-LiveTail.png)
+To manage data in Sumo Logic, do the following:
 
-To access additional Log Search, Metrics, and Live Tail options, do the following:
+1. From the top-right navigation bar, click **Configuration**.<br/><img src={useBaseUrl('img/get-started/config.png')} alt="img/get-started/config.png" width="400"/>
+1. Choose from the following, as needed:
+    * **Collection**. Manage [Collectors and Sources](/docs/send-data/collection), [OpenTelemetry Collection](/docs/send-data/opentelemetry-collector), [Data Collection Status](/docs/manage/ingestion-volume/collection-status-page), [Ingest Budget](/docs/manage/ingestion-volume/ingest-budgets), Archive, and [Data Archiving](/docs/manage/data-archiving).
+    * **Logs**. Manage [fields](/docs/manage/fields), [field extraction rules](/docs/manage/field-extractions), [parsers](/docs/search/search-query-language/parse-operators), [partitions](/docs/manage/partitions-data-tiers), [scheduled views](/docs/manage/scheduled-views), and [data forwarding](/docs/manage/data-forwarding).
```

Comment:
```suggestion
    * **Logs**. Manage [fields](/docs/manage/fields), [field extraction rules](/docs/manage/field-extractions), [parsers](/docs/cse/schema/parser-editor/), [partitions](/docs/manage/partitions-data-tiers), [scheduled views](/docs/manage/scheduled-views), and [data forwarding](/docs/manage/data-forwarding).
```

Reply:
@jpipkin1 , I think since this is in the Logs section, it's referring to log search parsers.